pyircbot/modules/StockPlay.py

```diff
@@ -143,10 +143,26 @@ def trader_background(self):
                         self.do_trade(data)
                     elif action == "portreport":
                         self.do_report(*data)
+                    elif action == "topten":
+                        self.do_topten(*data)
             except Exception:
                 traceback.print_exc()
                 continue
 
+    def do_topten(self, nick, replyto):
+        """
+        Do lookup of highest valued portfolios
+        """
+        self.log.warning("{} wants top 10 sent to {}".format(nick, replyto))
+
+        with closing(self.sql.getCursor()) as c:
+            for num, row in enumerate(c.execute("""SELECT h1.nick as nick, h1.total as total FROM stockplay_balance_history h1
+                                INNER JOIN (SELECT nick, max(day) as MaxDate FROM stockplay_balance_history GROUP BY nick) h2
+                                ON h1.nick = h2.nick AND h1.day = h2.MaxDate 
+                                ORDER BY total DESC LIMIT 10""").fetchall(), start=1):
```

Comment:
This query isn't working on my machine - `Error: no such column: h1.total`, on `SQLite version 3.24.0`.

We should also filter out the "dust" account here, since's it's internal and uses different units in the `balance` column than normal entries in this table. Its name in contained in the var `DUSTACCT`. 

Looks good otherwise!